Sources/UIColor+Sweet.swift

```diff
@@ -21,6 +21,17 @@ public extension UIColor {
         }
     }
 
+    /// Convenience initializers for RGB colors.
+    ///
+    /// - Parameters:
+    ///   - red: The red part.
+    ///   - green: The green part.
+    ///   - blue: The blue part.
+    ///   - alpha: The alpha part.
+    public convenience init(r red: Double, g green: Double, b blue: Double, a alpha: Double = 100) {
```

Comment:
The idea is that you should be able to drop the RGB values from here:

<img width="219" alt="915d2fd0-db3e-11e6-83c4-299e227b35dd" src="https://cloud.githubusercontent.com/assets/1088217/22286867/dbe75102-e2f0-11e6-9b62-d0edafea42ad.png">